specification/monitor/data-plane/Microsoft.Insights/preview/2023-05-01-preview/metricBatch.json

```diff
@@ -0,0 +1,535 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Monitor Metric Batch",
+    "version": "2023-05-01-preview"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{baseUrl}",
+    "useSchemePrefix": false,
+    "positionInOperation": "first",
+    "parameters": [
+      {
+        "name": "baseUrl",
+        "description": "The regional endpoint to use, for example https://eastus.metrics.monitor.azure.com. The region should match the region of the requested resources. For global resources, the region should be 'global'.",
+        "required": true,
+        "type": "string",
+        "in": "path",
+        "x-ms-skip-url-encoding": true
+      }
+    ]
+  },
```

Comment:
You should specify the security schemes and security requirements for your service -- something like this:
```
  "securityDefinitions": {
    "AzureAuth": {
      "type": "oauth2",
      "tokenUrl": "https://login.microsoftonline.com/common/oauth2/v2.0/token",
      "flow": "application",
      "description": "Azure Active Directory OAuth2 Authentication",
      "scopes": {
        "<audience>/.default": "client credential scope"
      }
    }
  },
  "security": [
    {
      "AzureAuth": [ "<audience>/.default" ]
    }
  ],
```

You will need to fill in the `<audience>` part with the correct audience for your service.